### PR TITLE
Fix _removeReaction's reaction lookup

### PR DESCRIPTION
**Pull request by fernozzle**
The `emojiID` that was used to remove a reaction from `this.reactions` was different from the `emojiID` that's used to add one.

`_removeReaction` would always return `null`, so the `messageReactionRemove` event could never get emitted. No more.

### CODE REVIEW

**Comment by fernozzle** [comment]
I had closed this because merging into master didn't seem right &mdash; 

&mdash; but since I can't see the indev branch and the contribution guide doesn't mention it, I am reopening this guy. Full steam ahead

**Comment by zajrik** [comment]
Yes, for future reference, master is now the dev branch. Stable is the release branch. From the announcement on the Discord.js Official Discord server:
>We've restructured the Git branches a bit. What used to be the indev branch is now master, and the old master is now stable. So, if you want to run the latest in-development version of discord.js, you should now simply use npm install hydrabolt/discord.js.

**Comment by GusCaplan** [comment]
instead of the build emoji thing, add a method to emojis which returns the correct string, maybe toEndpointString or something

File: src/structures/Message.js [hunk 1]
```diff
@@ -529,7 +529,7 @@ class Message {
   }
 
   _addReaction(emoji, user) {
-    const emojiID = emoji.id ? `${emoji.name}:${emoji.id}` : emoji.name;
+    const emojiID = buildEmojiID(emoji);
```
Comment (GusCaplan) [inline src/structures/Message.js]: change to emoji.identifier

File: src/structures/Message.js [hunk 2]
```diff
@@ -547,7 +547,7 @@ class Message {
   }
 
   _removeReaction(emoji, user) {
-    const emojiID = emoji.id || emoji;
+    const emojiID = buildEmojiID(emoji);
```
Comment (GusCaplan) [inline src/structures/Message.js]: change to emoji.identifier

File: src/structures/Message.js [hunk 3]
```diff
@@ -565,4 +565,8 @@ class Message {
   }
 }
 
+function buildEmojiID(emoji) {
```
Comment (GusCaplan) [inline src/structures/Message.js]: remove

**Pull request merged**